synapse/handlers/device.py

```diff
@@ -27,6 +29,21 @@ class DeviceHandler(BaseHandler):
     def __init__(self, hs):
         super(DeviceHandler, self).__init__(hs)
 
+        self.hs = hs
```

Comment:
It's kind of nice to kill of the ```self.hs``  as it means that all the accesses to the homeserver object happen during start up rather than during the execution.

Eh, the usage count is now about 50/50 so I guess it doesn't matter that much which style you pick:

```sh
$ grep -r self.hs.is_mine synapse | wc -l
34
$ grep -r self.is_mine synapse | wc -l
29
```